wrappers/services.go

```diff
@@ -288,3 +342,61 @@ WantedBy={{.ServicesTarget}}
 
 	return templateOut.Bytes()
 }
+
+func genSocketFile(appInfo *snap.AppInfo, socketName string) []byte {
```

Comment:
Let's please call this one genServiceSocketFile, to make it slightly more clear that this is not in fact an actual socket.

Reply:
Done